docs/integrations/builtin/cluster-nodes/sub-nodes/n8n-nodes-langchain.toolthink.md

```diff
@@ -0,0 +1,26 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+title: Think Tool node documentation
+description: Learn how to use the Think Tool node in n8n. Follow technical documentation to integrate the Tool Think node into your workflows.
+contentType: [integration, reference]
+priority: medium
+---
+
+# Think Tool node
+
+The Think Tool node is a [tool](/glossary.md#ai-tool) that invites an [agent](/glossary.md#ai-agent) to do some thinking before answering.
```

Comment:
It might be helful to include a bit more context for readers here. Just another sentence or two to expand on what this type of tool does and what circumstances they'd want to use it.

Reply:
Ah yeah, it looks like examples [aren't turned on in our config](https://github.com/n8n-io/n8n-docs/blob/main/mkdocs.yml#L117-L125) (and we use a different format for these that'd be closer to `/// example`).

I was able to get it working with some tweaks, but after seeing it rendered, I don't think that's necessarily the best formatting for this one (the example is a bit long for an admonition box and the block quotes clash a bit):

![image](https://github.com/user-attachments/assets/a07e473f-1793-45a0-a1e8-f6906c3cdb04)

I think I'm going to move it down to the existing examples section and tweak the formatting a bit. It's a really useful example though. Thanks for putting that together!

Reply:
A pleasure, let me know if you need me to do any more changes.